### PR TITLE
Map 16 extension

### LINKED ISSUE #115: Map/16 Extension of the map
As a user I would like the map to be bigger than it is now, especially it should fill every free space to the left and to the bottom of the screen (in desktop view), so I can see more in the same window. Also, no space is wasted. (marked light green)

IMG_A7CAEB698DE9-1

### PR DESCRIPTION
**Pull request by janina-adamcic**
closes #115

### CODE REVIEW

File: app/assets/javascripts/OutdoorMap/map_build.js [hunk 1]
```diff
@@ -4,7 +4,7 @@
 // Set the leaflet map with center and zoom-level
```
Comment (NikkelM) [inline app/assets/javascripts/OutdoorMap/map_build.js:4]: I noticed some unused imports on top of this file, maybe you could remove them, even though you did not introduce them.

File: app/views/map/index.html.erb [hunk 2]
```diff
@@ -1,23 +1,16 @@
...
-		<%= render 'indoor' %>
-	</div>
-  </body>
-</html>
\ No newline at end of file
+<%= content_tag :div, '', id: 'poi-data', data: { source: @points_of_interest } %>
+<%= javascript_include_tag 'OutdoorMap/geometry' %>
+
+<div class="leaflet" style="width: 100%; height: 100%;">
+	<div id="map" style="width: 100%; height: 100%;"></div>
```
Comment (NikkelM) [inline app/views/map/index.html.erb:13]: Regarding my comment with the picture: I suppose this `div` not covering the whole page, excluding the navigation bar on the top, is what limits the "fullscreen"?

**Review by NikkelM — DISMISSED** [review]
The are still some white borders on the sides and the top of the page - it would be great if the map would fill those parts as well. Is that possible? Should this be a priority @blackjack2693 ? 
image
The map definitely looks a lot nicer though :)

File: app/views/map/index.html.erb (hunk 2 above)
Comment (janina-adamcic) [reply app/views/map/index.html.erb:13]: No. The reason for the white borders around the map was some padding and margin specified in a higher level stylesheet. I can override this padding and margin though, the map indeed looks better without it.

Comment (NikkelM) [reply app/views/map/index.html.erb:13]: Looks great! I think now that it is "fullscreen mode" anyways, we could remove the fullscreen-handle, as that maybe clutters the interface a bit, especially on mobile where there is less space. What's your opinion on that?

Comment (felix-20) [reply app/views/map/index.html.erb:9]: fullscreen has been removed in a05f1bef4147c5720b9f86db42956da2cf8ad797

File: app/assets/javascripts/OutdoorMap/map_build.js [hunk 3]
```diff
@@ -1,10 +1,7 @@
-// import L from 'leaflet';
-// import 'leaflet.locatecontrol';
-
 // Set the leaflet map with center and zoom-level
 const standardZoomLevel = 17;
 const indoorZoomLevel = 19;
-var mymap = L.map('map').setView([52.39300, 13.12900], standardZoomLevel);
+var mymap = L.map('map', {fullscreenControl: true}).setView([52.39300, 13.12900], standardZoomLevel);
```
Comment (NikkelM) [inline app/assets/javascripts/OutdoorMap/map_build.js:4]: I assume we can remove the `fullscreenControl` here as well?

**Review by NikkelM — APPROVED** [review]

**Review by NikkelM — APPROVED** [review]

**Pull request merged**